### ISSUE TITLE
Add fleet level overrides for rainbow splash screen, GPU memory etc

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/helium-miner-software#303]
Curently we have this set in the config.txt on raspberry pi - but we should set fleet-wide overrides for this to make sure it is properly set on all devices

**Comment by shawaj** [comment]
somewhat related to #164

**Comment by shawaj** [comment]
@MuratUrsavas this is kind of similar to what we were discussing earlier today

**Comment by MuratUrsavas** [comment]
@shawaj you're right. We can pull this to the next sprint as our hardware vendor count increases. And from the test results of our software stack, I don't think it would stop anytime soon :wink:

**Comment by shawaj** [comment]
Related to #380

**Comment by shawaj** [comment]
Think I've sorted this now but need to have a run through it and triple check it's all correct as there are quite a lot of fleets now
